### PR TITLE
Data model changes issue279

### PR DESCRIPTION
**Pull request by thutuva**
## Describe your changes
Please include a summary of any changes.
Added the following columns
- payer (medical and pharmacy)
- plan (medical, eligibility, pharmacy)
- coinsurance_amount (medical and pharmacy)
- copayment_amount (medical and pharmacy)
- deductible_amount (medical and pharmacy)

## How has this been tested?
Please describe the tests you ran to verify your changes.  Provide instructions or code to reproduce output.
CI testing

## Reviewer focus
Please summarize the specific items you’d like the reviewer(s) to look into.
N/A

## Checklist before requesting a review
- [ ]  (Optional) I have recorded a Loom performing a self-review of my code
- [ ]  My code follows style guidelines
- [ ]  I have commented my code as necessary
- [ ]  (New models only) I have implemented generic dbt tests to validate primary keys/uniqueness in my model
- [ ]  I have added at least one Github label to this PR

### CODE REVIEW

File: dbt_project.yml [hunk 1]
```diff
@@ -7,8 +7,8 @@ profile: default
 vars:
 
   ## Set the database and schema where raw CCLF data is stored
-  input_database: default
-  input_schema: default
```
Comment (sarah-tuva) [inline dbt_project.yml:11]: Was this on purpose or just left over from testing?

File: packages.yml [hunk 2]
```diff
@@ -2,4 +2,4 @@ packages:
   - package: dbt-labs/dbt_utils
     version: [">=0.9.2","<1.0.0"]
   - package: tuva-health/the_tuva_project
-    version: [">=0.5.0","<1.0.0"]
```
Comment (sarah-tuva) [inline packages.yml:5]: This version range looks wrong. I think it should be [">=0.6.0","<0.7.0"]

Comment (thutuva) [reply packages.yml:5]: @sarah-tuva this comment is on the 'before' diff.  The package version has been correctly updated.

File: dbt_project.yml (hunk 1 above)
Comment (thutuva) [reply dbt_project.yml:11]: @sarah-tuva this comment is on the 'before' diff.   The dbt_project.yml currently points to CI which I think is okay.  I don't want to rerun CI and the schema still says CCLF

**Review by tuvaforrest — APPROVED** [review]

**Pull request merged**